src/js/events.js

```diff
@@ -5,6 +5,120 @@
  * robust as jquery's, so there's probably some differences.
  */
 
+import * as Lib from './lib';
+import window from 'global/window';
+import document from 'global/document';
+
+/**
+ * Fix a native event to have standard property values
+ * @param  {Object} event Event object to fix
+ * @return {Object}
+ * @private
+ */
+var fixEvent = function(event) {
+
+  function returnTrue() { return true; }
+  function returnFalse() { return false; }
```

Comment:
@dmlap is just saying define them in the module scope instead of each time the fixEvent function is run. They could be moved out now.